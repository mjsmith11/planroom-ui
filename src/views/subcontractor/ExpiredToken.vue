<template>
    <div class="container">
        <logo-header></logo-header>
        <div class="card message">
            <div class="card-body">
                <div class="card-text">
                    We're sorry. Your link has expired. Please contact our office if you still require access.
                </div>
              </div>
        </div>
        <contact-info></contact-info>
    </div>
</template>
<script>
import LogoHeader from '../../components/LogoHeader'
import ContactInfo from '../../components/ContactInfo'
export default {
  components: {
    logoHeader: LogoHeader,
    contactInfo: ContactInfo
  }
}
</script>
<style scoped>
    .message{
        width: 60%;
        margin: auto;
        text-align: center;
    }
</style>
