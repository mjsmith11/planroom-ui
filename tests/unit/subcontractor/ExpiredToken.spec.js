import { mount } from '@vue/test-utils'
import ExpiredToken from '@/views/subcontractor/ExpiredToken'

describe('Expired Token', () => {
  it('has correct html structure', () => {
    const cmp = mount(ExpiredToken, {
      stubs: [
        'logo-header',
        'contact-info'
      ]
    })
    expect(cmp.html()).toMatchSnapshot()
  })
})
