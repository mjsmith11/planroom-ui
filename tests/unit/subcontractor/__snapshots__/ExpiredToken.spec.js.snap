// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expired Token has correct html structure 1`] = `
<div class="container">
  <logo-header-stub></logo-header-stub>
  <div class="card message">
    <div class="card-body">
      <div class="card-text">
        We're sorry. Your link has expired. Please contact our office if you still require access.
      </div>
    </div>
  </div>
  <contact-info-stub></contact-info-stub>
</div>
`;
